packages/jest-preset-cli/src/config.ts

```diff
@@ -0,0 +1,10 @@
+import path from 'path';
+import readPkg from 'read-pkg';
+
+export function getConfig(cwd: string) {
+  return {
+    preset: '@expo/jest-preset-cli',
+    rootDir: path.resolve(cwd),
+    displayName: readPkg.sync({ cwd }).name,
+  };
```

Comment:
Sounds good! I removed the abstraction 😄 The `rootDir` is still a necessity though. If you run the tests from the package folder, it works without the `rootDir` because Jest can easily resolve it. But, if you run this from root ([using the projects config](https://github.com/expo/expo-cli/blob/master/jest.config.js#L2-L11)), these root dirs are automatically resolved to the root of the project. This causes some issues with "duplicate mocks".